readme.md

```diff
@@ -306,6 +306,7 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[@foo-software/lighthouse-persist](https://github.com/foo-software/lighthouse-persist)** - Run Lighthouse and upload HTML reports to an AWS S3 account.
```

Comment:
Using the scoped package name as it [appears on NPM](https://www.npmjs.com/package/@foo-software/lighthouse-persist) since theoretically another package could be introduced with the same name. Please let me know if you think it should simply be `lighthouse-persist` and I’ll change it.

Reply:
Yeah I would just put it as `lighthouse-persist` and then move it in-between `lighthouse-mocha-example` and `lighthouse4u`.

Does it make sense to call it an S3 account?  Afaik it is an _AWS account_ with _S3 Buckets_, not really an S3 account?
```markdown
* **[lighthouse-persist](https://github.com/foo-software/lighthouse-persist)** - Run Lighthouse and upload HTML reports to AWS S3 buckets.
```

Reply:
Thanks @exterkamp - good point. Updated in fe554f4.